### PR TITLE
airTable fetching is added

### PR DESCRIPTION
**Pull request by ozzydogandev**
-Added function for fetching data from Airtable
-Added function to create a todo item in Airtable
-Added function to remove a todo item from Airtable
-Removed useEffect contents except localStorage (?)
-Removed addTodo and removeTodo functions and redefined them under functions useEffect functions for POST and DELETE method

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -8,32 +8,105 @@ function App() {
...
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${import.meta.env.VITE_AIRTABLE_API_TOKEN}`,
+      },
+    };
+
+    try{
+      const response = await fetch(url, options);
+      if (!response.ok) {
+        throw new error(`Error: ${response.status}`);
```
Comment (AnnaPestova1) [inline src/App.jsx:23]: class Error writes with capital letter

File: src/App.jsx [hunk 2]
```diff
@@ -8,32 +8,105 @@ function App() {
...
+      }
+
+      const newTodo = await response.json();
+      setTodoList([...todoList, { id: newTodo.id, title: newTodo.fields.title }]);
+    } catch (error) {
+      console.error(error.message);
+    }
+  };
+
+  const addTodo = (newTodo) => {
```
Comment (AnnaPestova1) [inline src/App.jsx]: why do not you use  postTodo() function, but put this single function inside addTodo() function?

File: src/App.jsx [hunk 3]
```diff
@@ -8,32 +8,105 @@ function App() {
...
+      }
+
+      const updatedTodoList = todoList.filter((todo) => todo.id !== id);
+      setTodoList(updatedTodoList);
+    } catch (error) {
+      console.error(error.message);
+    }
+  };
+
   useEffect(() => {
```
Comment (AnnaPestova1) [inline src/App.jsx]: You do not need this useEffect anymore. Now you interact with airtable

**Review by AnnaPestova1 — APPROVED** [review]
Great work Oguzhan!
Check the comments please. And you have one small issue regarding id creating with Date now:
read the assignment
image
you can fix this issue, it will help you to have clear code on your final project .

PS do not forget to fix typo in the word Error in Error class

**Comment by ozzydogandev** [comment]
Hi @AnnaPestova1 ! Thank you so much for reviewing my work. I just fixed the issues and pushed my changes for your review. I know I'm catching everything late but I'm doing my best to catch the class and finish the class on time. A lot of things happened back to back in my personal life. But I will catch the class and submit my all assignments this week. 🙏🏻

**Pull request merged**